### PR TITLE
Added navigation between @OnClick and @InjectView.

### PR DESCRIPTION
**Pull request by Ghedeon**
Navigation functionality similar to Dagger/Otto square IntelliJ plugins. Creates clickable marker that links @OnClick method and @InjectView field for the same resource id. Especially useful in a big files with complex layout due to immediate auto scroll to the right place.

### CODE REVIEW

File: META-INF/plugin.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <idea-plugin version="2">
 	<id>eu.inmite.android.plugin.butterknifezelezny</id>
 	<name>Android ButterKnife Zelezny</name>
-	<version>1.2</version>
+	<version>1.3</version>
```
Comment (Ghedeon) [inline META-INF/plugin.xml]: not quite sure how big this change is. Increased minor version so far.

File: META-INF/plugin.xml [hunk 2]
```diff
@@ -48,7 +53,7 @@
 		<action id="AddInjectionsAction"
 			class="com.avast.android.butterknifezelezny.InjectAction"
 			text="Generate _Butterknife Injections"
-			icon="/com/avast/android/butterknifezelezny/icons/butterknife-zelezny.png"
```
Comment (Ghedeon) [inline META-INF/plugin.xml]: moved plugin icon to the dedicated resources folder. Perhaps, you'll need to perform "right click -> mark as resource folder" in your IDE (alternatively can be done via project structure dialog).

**Comment by destil** [comment]
Thanks for your contribution! It's a nice feature and your code is OK.

Just please change the version to 1.3-SNAPSHOT since it's unreleased version yet.

**Comment by Ghedeon** [comment]
done

**Pull request merged**